### PR TITLE
[SYTO-1176] Remove color tints from Mortar

### PR DESCRIPTION
**Pull request by hkekeli**
https://jira.natgeo.com/browse/SYTO-1176

Confirm `affirmation-l` and `danger-l` are both removed from Mortar's codebase and documentation. 

The color page should resemble the mock. You can test here: http://assets-qa.int.ngeo.com/mortar/dev/color/

### CODE REVIEW

**Comment by pmourelle** [comment]
There're still definition and uses of `affirmation-l` in code:
- src/styles/patterns/_buttons.scss
- src/styles/themes/default/vars/_colors.scss

**Comment by hkekeli** [comment]
Done!

File: src/styles/themes/default/vars/_colors.scss [hunk 1]
```diff
@@ -53,9 +53,6 @@ $mt3_social-colors: (
 // Affirmation
 $mt3_color-affirmation-l: mt3_tint($mt3_color-affirmation, 25);
```
Comment (pmourelle) [inline src/styles/themes/default/vars/_colors.scss]: Hi there :)

Comment (hkekeli) [inline src/styles/themes/default/vars/_colors.scss]: haha

**Pull request merged**